project/src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps;
+
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class UserTest {
+  private static final String NAME = "Bob Smith";
+  private static final String EMAIL = "bobsmith@example.com";
+  private static final Set<String> INTERESTS = new HashSet<>(Arrays.asList("Conservation", "Food"));
+  private static final Set<String> NEW_INTERESTS =
+      new HashSet<>(Arrays.asList("Conservation", "Food", "Music"));
+  private static final Set<String> SKILLS = new HashSet<>(Arrays.asList("Cooking"));
```

Comment:
I think you can use ImmutableList.of here from guava.

Reply:
Done